README.md

```diff
@@ -285,6 +286,38 @@ This library also exposes functions for encoding JavaScript values into Uint8Arr
 
 Remember, it is not required to use these encoding functions. The JavaScript code must only return a [Uint8Array](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Uint8Array) which represents the `bytes` that are returned on-chain.
 
+### Importing Dependencies
+
+To import and use libraries in your Functions request JavaScript source code, you must use the async `import` function. Since this is an async function, you must remember to use the `await` keyword to wait for the dependency to be imported before it can be used as shown in the examples below.
+
+```
+const lodash = await import("http://cdn.skypack.dev/lodash");
+const result = lodash.concat([1], 2);
+return Functions.encodeString(JSON.stringify(result));
+```
+
+```
+const { ethers } = await import("npm:ethers@6.9.0");
+const myNumber = ethers.AbiCoder.defaultAbiCoder().decode(
+  ["uint256"],
+  "0x000000000000000000000000000000000000000000000000000000000000002a"
+);
+return Functions.encodeUint256(BigInt(myNumber.toString()));
+```
+
+> ⚠️ **Users are fully responsible for any dependencies their JavaScript source code imports. Chainlink is not responsible for any imported dependencies and provides no guarantees of the validity, availability or security of any libraries a user chooses to import. Developers are advised to fully vet any imported dependencies or avoid dependencies altogether to avoid any risks associated with a compromised library or a compromised repository from which the dependency is downloaded.**
+
+Chainlink Functions supports importing ESM-compatible modules with are supported by Deno within the JavaScript source code. It also supports importing some NPM packages [via the `npm:` specifier](https://docs.deno.com/runtime/manual/node/npm_specifiers) and some built-in Node.js modules [via the `node:` specifier](https://docs.deno.com/runtime/manual/node/node_specifiers). Check out the [Deno documentation on importing modules](https://docs.deno.com/runtime/manual/basics/modules/) for more information or visit [deno.land/x](https://deno.land/x) to find 3rd party modules which have been built for Deno.
```

Comment:
nit: is Nodejs standard library a better way to indicate "built in"?  I'm divided on this so either way works.